packages/sealed_countries/CHANGELOG.md

```
@@ -1,3 +1,20 @@
+## 2.2.0
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)